index.html

```diff
@@ -10,23 +10,215 @@
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+    <div class="project" id="project2">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+     <div class="project" id="project3">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Tonic</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+    <div class="project" id="project4">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>  
+  </section>
+
+  <section class="about-me">
+    <div class="primary-text-container">
+      <h1 class="primary-text">About<br>Myself</h1>
```

Comment:
- [ ] I suggest removing the `<h1>` tag from here and using another header tag `hx` (eg `<h2>`,  `<h3>`, ...) instead. to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> Note: you used `<h1>` here ``` <h1 class="primary-text">I'm Juan Diaz Glad to see you!</h1>```, that's great because this is the main title 💯